torch/onnx/_internal/fx/op_validation.py

```diff
@@ -165,8 +182,28 @@ def validate_op_between_ort_torch(
                 diagnostic.level = diagnostics.levels.WARNING
 
 
+@_beartype.beartype
+def _convert_symint_to_int_in_shape(shape: torch.Size) -> torch.Size:
```

Comment:
```suggestion
def _try_convert_symint_to_int_in_shape(shape: torch.Size) -> torch.Size:
```
This may or may not convert a symint, so a `try` might be more accurate? not a strong preference though

Reply:
it raises when it cannot, so maybe add `raises` into docstring and keep the name? Usually we name with `_try` or `_maybe` if it gives up without raising.

Reply:
Keep the name and added docstring